XRFoundation/XREngine
# Refactor, drive wallet login off of admin settings

## Summary

Refactors login with wallet flow.
Drives wallet admin off admin settings vs env flag.

## References

Part of #6584.

## QA Steps

Go to Admin > Settings > Auth, turn on DidWallet toggle, reload, then profile menu should display 'Login with Wallet' button.

Toggle off, reload, profile should /not/ have the login with wallet button.
